source/hassio/HassOS-FAQ.markdown

```diff
@@ -0,0 +1,60 @@
+---
+title: "Home Assistant OS FAQ"
+description: "Frequently Asked Questions for HA OS"
+---
+
+# Home Assistant OS FAQ
+
+## Is USB Boot for the Pi4 supported? Is the Pi4 with 8GB RAM supported?
+
+The bootloader for Home Assistant OS (uboot) does not yet support Pi4 booting from USB. Support is likely a month or two away as of July 2020.
+
+The Pi4 which comes with 8GB RAM is similarly waiting for support from upstream.
+
+
+## How do I run a specific version of Home Assistant?
+   
+For this you would need to install the [Terminal & SSH add-on][ssh] or use the console
+that is available on your device by connecting a keyboard and screen.
+
+To install the Terminal & SSH add-on, choose **Supervisor**, which is located in the sidebar and then the add-on store.
```

Comment:
```suggestion
To install the Terminal & SSH add-on, choose **Supervisor**, which is located in the sidebar and then the add-on store. If you don't see it, enable "Advanced Mode" from your profile page.
```
This add-on won't be visible by default in standard mode, so requires further action for some users.